.azure-pipelines.yml

```diff
@@ -0,0 +1,130 @@
+#
+# Azure Pipelines configuration for building and testing create-react-app on Linux, Windows, and macOS.
+#
+
+trigger:
+- master
+
+variables:
+  CI: true
+  VSTS_OVERWRITE_TEMP: True

```

Comment:
Setting `VSTS_OVERWRITE_TEMP` avoids test problems related to weirdness with "temp" path math (especially on Windows where the short and long file paths are different)